README.md

```diff
@@ -74,6 +74,25 @@ $ npm run build.dev -- --env.nolint
 $ npm run docs
 ```
 
+## Transifex Client
+Run transifex client (tx) in your local synchronized 'develop' branch
+Transifex client behavior is determined by .tx/config file in the repo.
+```sh
+# Install Transifex Client (tx)
+$ pip install transifex-client
+# Check if install was successful
```

Comment:
@zarembsky looks like there's a `tx init` command that needs to be run to set up the tx command line tool. Please add instructions on how to set this up.